<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatternPropertiesDialog_UI</class>
 <widget class="QDialog" name="PatternPropertiesDialog_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="TextLabel1">
         <property name="text">
          <string>New Pattern Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="patternNameTxt"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Pattern description</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="patternDescTxt"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Pattern category</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="categoryComboBox">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>37</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelBtn">
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
         <property name="shortcut">
          <string>Alt+C</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="okBtn">
         <property name="text">
          <string>&amp;OK</string>
         </property>
         <property name="shortcut">
          <string>Alt+O</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>37</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
